rpc/tests/assert_helpers.rs

```diff
@@ -166,9 +166,18 @@ pub fn assert_transaction(ether_tx: Transaction, starknet_tx: StarknetTransactio
                         starknet_address_to_ethereum_address(&v0.contract_address)
                     );
                     let signature = Signature {
-                        v: felt_option_to_u256(Some(&v0.signature[2])).unwrap(),
-                        r: felt_option_to_u256(Some(&v0.signature[0])).unwrap(),
-                        s: felt_option_to_u256(Some(&v0.signature[1])).unwrap(),
+                        r: U256::from_str(
+                            "0x32506f889d51447e4c523f744913a358fcfe45a3dbf624a9e63bd9f312801528",
```

Comment:
How did you obtain this signature? To what payload is it linked?

Reply:
This signature was obtained from the first transaction in the `starknet_getBlockWithTxs.json` file. I manually extracted it from the calldata. I could decode it from using the `decode_signature_from_tx_calldata` helper but I felt like this was redundant and not properly testing the function. Let me know what you think.

Reply:
I think it's good, I noticed on Madara as well that we use hardcoded signatures in tests which is good, but don't provide enough comments above about where the sig came from. I think it'd be a good practice to enforce to comment above each hardcoded hash or signature with its inputs values, where it comes from etc. even if the comment is multi-line or verbose, it tremendously helps.